core/src/main/java/org/apache/gravitino/metalake/MetalakeManager.java

```diff
@@ -86,8 +89,8 @@ public class MetalakeManager implements MetalakeDispatcher {
                           .build())))
           .build();
 
-  @VisibleForTesting
-  public static void clearCache() {
+  @Override
+  public void close() throws IOException {
```

Comment:
need call `close` in there:
https://github.com/apache/gravitino/blob/0e41493414fef118187c5e88c5ae75aca4bcb064/core/src/main/java/org/apache/gravitino/GravitinoEnv.java#L362